UnitsNet.Extensions/UnitsNet.Extensions.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- NuGet properties -->
+  <PropertyGroup>
+    <PackageId>UnitsNet.Extensions</PackageId>
+    <Version>1.0.0</Version>
+    <Authors>Andreas Gullberg Larsen</Authors>
+    <Title>Units.NET Extensions</Title>
+    <Description>A extension library for Units.NET.</Description>
```

Comment:
Description could be a bit more descriptive.
How about `Adds convenient extension methods to Units.NET, such as 5.Meters().` ?

Reply:
How about "Auxiliary functionality for Units.NET" or something like that? I'm not sure I love the example (5.Meters()), as the idea here (at least the way I understand it) is to put any relevant (auxiliary) functionality in this package - shouldn't be limited to the "numbers to" extensions.